frontend/tests/App.spec.js

```diff
@@ -59,6 +60,73 @@ describe('App.vue', () => {
     expect(wrapper.vm.collections).toEqual("collection");
   })
 
+  it('changeItemsState - method', () => {
```

Comment:
the BDD style ("describe"/"it"/...) should tell a **story**, more specifically 
it should act as a **specification**

See example: https://mochajs.org/#bdd

Try to improve the descriptions of your `it` (tests) calls

Reply:
for example this test inspect that a **nested** item (1 level) can be modified.